app/src/main/res/layout/fragment_memory_graph.xml

```diff
@@ -6,5 +6,16 @@
     <com.github.mikephil.charting.charts.LineChart
         android:id="@+id/mem_graph"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:clipChildren="true"
+        android:addStatesFromChildren="false">
+
+        <ProgressBar
```

Comment:
This is the standard progress bar: https://developer.android.com/reference/android/widget/ProgressBar.html
We'll use it as a dialog. Current one is soo big.

Reply:
Yes actually :joy: I tried with lots of layout changes but the progress bar won't come as overlay on top.